typescriptFixme.ts

```diff
@@ -107,9 +138,9 @@ class Car {
 }
 
 // annotate this array
-var arrayOfAnyType = [1, "string", false];
+var arrayOfAnyType: any[] = [1, "string", false];
```

Comment:
don't use `any`. Specify the possible types this array can have

Reply:
var arrayOfAnyType: (number | string | boolean) [ ] = [1, "string", false];

Reply:
this is correct. commit this change and push it up